os/binfmt/Kconfig

```diff
@@ -90,6 +90,18 @@ config OPTIMIZE_APP_RELOAD_TIME
 		the MPU requirements. Hence, using this option can result in a steep increase in the memory
 		requirement for this application.
 
+config BINFMT_SECTION_UNIFIED_MEMORY
+	bool "Allocate section memory as one chunk"
+	depends on OPTIMIZE_APP_RELOAD_TIME
+	default n
+	---help---
+		When this option is enabled, binary loader allocates one big memory chunk enough to contains each loading sections
```

Comment:
Need to change `contains` to `contain`

Reply:
Why? The subject of contains is `one big memory chunk`, and it is the singular.